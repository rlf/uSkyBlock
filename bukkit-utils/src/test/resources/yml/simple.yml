#
# This is a simple Yml file
#
# with multiple comments
root:
  child-value: 10

  # child nodes
  child node:
      some-double: 1.34  # a number
      some-section:
        section-list: [Hej, Mor]
        another-list:
          - what now
          - do you know?
      # this is a key and a value
      key: 'value'
  another node:
    enabled: true
  # a comment
  a.section.deeper:
    b: # b comment
      c: false # should be false
"an.annoying:key this.is:for.sure": hello world