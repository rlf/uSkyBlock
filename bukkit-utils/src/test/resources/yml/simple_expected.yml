#
# This is a simple Yml file
#
# with multiple comments
root:
  child-value: 10

  # child nodes
  child node:

    # a number
    some-double: 1.34
    some-section:
      section-list:
      - Hej
      - Mor
      another-list:
      - what now
      - do you know?

    # this is a key and a value
    key: value
    abe: |-
      lincoln
      was a wonderful
      president
  another node:
    enabled: true
  a:
    section:

      # a comment
      deeper:

        # b comment
        b:

          # should be false
          c: false
an:
  annoying:key this:
    is:for:
      sure: hello world
